src/benchmarks/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
 ##################################
 # Benchmark executable
 
-add_executable(end_to_end_bench src/end_to_end_bench.cpp)
+add_executable(end_to_end_bench src/end_to_end_bench.cpp src/bench_utils.cpp src/include/bench_utils.hpp)
```

Comment:
Put the `bench_utils.hpp` file in `src/`. In this case, nothing outside of `src/benchmarks/src` should need to include `bench_utils` so it's cleaner to just keep everything in one directory.